cpp/oneapi/dal/backend/primitives/stat/cov.hpp

```diff
@@ -38,11 +73,29 @@ sycl::event correlation(sycl::queue& queue,
                         const ndview<Float, 2>& data,
                         const ndview<Float, 1>& sums,
                         ndview<Float, 2>& corr,
-                        ndview<Float, 1>& means,
+                        const ndview<Float, 1>& means,
```

Comment:
We have an agreement that we pass immutable objects first, then mutable ones. Please change order.